test/runtests.jl

```diff
@@ -33,7 +35,7 @@ MOIU.@model(Model,
              MOI.PositiveSemidefiniteConeTriangle, MOI.PositiveSemidefiniteConeSquare,
              MOI.RootDetConeTriangle, MOI.RootDetConeSquare, MOI.LogDetConeTriangle,
              MOI.LogDetConeSquare),
-            (MOI.PowerCone, MOI.DualPowerCone, MOI.SOS1, MOI.SOS2),
+            (MOI.PowerCone, MOI.DualPowerCone, MOI.SOS1, MOI.SOS2, LessThanIndicatorSet),
```

Comment:
Is something lost by supporting `MOI.IndicatorSet` directly? Someone reading this code will immediately wonder why the choice of `LessThanIndicatorSet`.

Reply:
When creating a `Model{T}`, sets `S` in the previous category will be typed `S` and sets in this category will be typed `S{T}`. Here `MOI.IndicatorSet` needs to be typed `MOI.IndicatorSet{MOI.ACTIVATE_ON_ONE, MOI.LessThan{T}}` so it does not fall into any category while `LessThanIndicatorSet` falls in the second category.
We could use `MOI.IndicatorSet{MOI.ACTIVATE_ON_ONE}` while could work (untested) but the container will have a non-concrete type.

Reply:
so this one is resolved?